packages/ng/src/app/select/option/select-option-feeder.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+	Component,
+} from '@angular/core';
+import {ISelectOptionFeeder} from './select-option-feeder.model';
+
+/**
+ * The component that provides available options for lu-select
+ */
+@Component({
+	template: '',
+})
+export class AbstractSelectOptionFeederComponent<T> implements ISelectOptionFeeder<T> {
```

Comment:
calling it abstract doesn't make it abstract

Reply:
https://github.com/LuccaSA/lucca-front/pull/225#discussion_r172152209